Formula/cql.rb

```diff
@@ -0,0 +1,23 @@
+class Cql < Formula
+  desc "Decentralized SQL database with blockchain features"
+  homepage "https://covenantsql.io"
+  url "https://github.com/CovenantSQL/CovenantSQL/archive/v0.5.0.tar.gz"
+  sha256 "ebee3e8fca672d6f3d3d607e380ec7a1f3a31501e5c80b85c660f50eda7a2240"
+  head "https://github.com/CovenantSQL/CovenantSQL.git"
+
+  depends_on "go" => :build
+  depends_on "make" => :build
```

Comment:
Does this _really_ need the latest GNU make? If it can build with macOS make, please remove this line.

Reply:
Fixed